### ISSUE TITLE
python3 summarize flag bug

### ISSUE THREAD

**Issue by kensley** [catkin/catkin_tools#510]
### System Info

* Operating System: Ubuntu:16.04 Docker container, host kernel: `Linux 977d91afe8d8 4.4.0-112-generic #135-Ubuntu SMP Fri Jan 19 11:48:36 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux`
* Python Version: `3.5.2`
* Version of catkin_tools: `pip3 freeze | grep catkin-tools
catkin-tools==0.4.4`
* ROS Distro: `kinetic`

### Issue

When running the following command: 
`catkin build my_pkg --no-deps --summarize`

I get the following output in the end:

```bash
Finished  <<< my_pkg                               [ 6.1 seconds ]                                                                                                                                                                                              
                                                                                                                                                                                                                                                                
[build] Successful packages:                                                                                                                                                                                                                                    
                                                                                                                                                                                                                                                                
Exception in thread Thread-1:
Traceback (most recent call last):
  File "/usr/lib/python3.5/threading.py", line 914, in _bootstrap_inner
    self.run()
  File "/usr/local/lib/python3.5/dist-packages/catkin_tools/execution/controllers.py", line 748, in run
    self.print_exec_summary(completed_jobs, warned_jobs, failed_jobs)
  File "/usr/local/lib/python3.5/dist-packages/catkin_tools/execution/controllers.py", line 322, in print_exec_summary
    sorted(successfuls.items() + ignoreds.items()),
TypeError: unsupported operand type(s) for +: 'dict_items' and 'dict_items'

[build] Note: Workspace packages have changed, please re-source setup files to use them.
```

I'm guessing because python2 returns `.items()` as a list you add them, but in python3, `.items()` are view objects.

**Comment by kensley** [comment]
Ugly but quick hack:
`sorted(list(successfuls.items()) + list(ignoreds.items()))`

**Comment by timonegk** [comment]
This would be closed by #565.

**Comment by timonegk** [comment]
This issue was fixed in #565 and can be closed.
